src/components/Modal/styles.ts

```diff
@@ -226,10 +226,10 @@ export const ModalWrapper = styled(Modal)`
   width: ${rem(300)};
 `;
 
-export const KycAndFundsBack = styled.div`
+export const CoveringBack = styled.div`
```

Comment:
We already have Backdrop Component(see Backdrop.tsx). I think we should leave only one of this 2 components(I mean CoveringBack or Backdrop)